cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -169,7 +169,7 @@ private void checkServicesWithRetry(Callable<Boolean> callable,
           throw Throwables.propagate(rootCause);
         }
       }
-    } while (numSecs <= SERVICE_CHECK_TIMEOUT);
+    } while (sw.elapsedTime(TimeUnit.SECONDS) <= SERVICE_CHECK_TIMEOUT_SECONDS);
 
     // when we have passed the timeout and the check for services is not successful
     throw new TimeoutException(exceptionMessage);
```

Comment:
hmm.. when a test took a long time, getting that info that retry for services took 10 mins would be helpful i guess.